db/re/item_db_equip.yml

```diff
@@ -72049,6 +72345,217 @@ Body:
          bonus bAllStats,1;
          bonus2 bExpAddRace,RC_All,10;
       }
+  - Id: 15440
+    AegisName: Fierce_Physique_Armor_E
+    Name: Fierce Physique Armor
+    Type: Armor
+    Defense: 50
+    Slots: 1
+    Locations:
+      Armor: true
+    ArmorLevel: 1
+    EquipLevelMin: 70
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bBaseAtk,5;
+      bonus bStr,3;
+      bonus bDex,3;
+      if (.@r>=5) {
+         bonus bFlee,5;
+         bonus bDef,5;
+      }
+      if (.@r>=7) {
+         bonus bDef,10;
+         bonus bMdef,2;
+      }
+      if (.@r>=9) {
+         bonus bDef,15;
+         bonus bMdef,3;
+      }
+      if (.@r>=10) {
+         bonus bBaseAtk,10;
+         bonus bMdef,5;
+         bonus2 bSubEle,Ele_Earth,2;
+         bonus2 bSubEle,Ele_Fire,2;
+         bonus2 bSubEle,Ele_Water,2;
+         bonus2 bSubEle,Ele_Wind,2;
+         bonus2 bSubRace,RC_DemiHuman,1;
+         bonus2 bSubRace,RC_Player_Human,1;
+         bonus2 bSubRace,RC_Player_Doram,1;
+         if (BaseClass == Job_Swordman || BaseClass == Job_Merchant)
+             bonus bMaxHP,600;
+         else if (BaseClass == Job_Archer || BaseClass == Job_Thief || BaseClass == Job_Gunslinger || (BaseJob == Job_Taekwon && Class!=Job_Star_Gladiator))
```

Comment:
Correct me but `BaseJob == Job_Taekwon && Class!=Job_Star_Gladiator` seems wrong, for me Job_Taekwon is a BaseClass. I will check later

Reply:
```
message strcharinfo(0),"BaseJob: "+BaseJob+" / BaseClass: "+BaseClass+" / Class: "+Class;
```
![Capture](https://github.com/rathena/rathena/assets/60462533/dc54981b-8be0-42ad-a772-30d92d0473c0)